src/main/java/sparksoniq/jsoniq/runtime/iterator/functions/DynamicFunctionCallIterator.java

```diff
@@ -123,6 +125,7 @@ private void setFunctionItemAndIteratorWithCurrentContext() {
         }
         _functionCallIterator = Functions.buildUserDefinedFunctionCallIterator(
             _functionItem,
+            _functionItem.getBodyIterator().getHighestExecutionMode(),
```

Comment:
For dynamic call, I think this is always local no matter what.

(only static UDF calls are able to execute the function body as an RDD).

Reply:
DynamicFunctionCallRuntimeIterator always evaluates locally indeed. However if a UDF is RDD/DF enabled, I think it could be evaluated in that manner. In this scope of this PR all UDFs are locally evaluated indeed. But #451 will be bringing the support for RDD/DF enabled UDFs.
For the example below,  statically detected postFixExtension of dynamicFunctionCall will be locally executed. However the function invoked by this dynamicFunctionCall could be evaluated with RDDs. Please correct me if there is something that I am missing.

declare function test() as integer{
    parallelize((1,2,3))
};
test#0()